src/main/java/mate/academy/spring/dto/request/CinemaHallRequestDto.java

```diff
@@ -1,7 +1,12 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.Min;
+import javax.validation.constraints.Size;
+
 public class CinemaHallRequestDto {
+    @Min(50)
     private int capacity;
```

Comment:
please use objects instead of primitives in DTOs for better representation of missing data